roles/kubernetes/kubeadm_common/defaults/main.yml

```diff
@@ -20,4 +20,5 @@ kubeadm_patches: []
 # Patches are applied in the order they are specified.
 
 # List of errors to ignore during kubeadm preflight checks
-kubeadm_ignore_preflight_errors: []
+kubeadm_ignore_preflight_errors:
+- Port-10250
```

Comment:
This can't be done here because this would break customization of this variable.
It should instead be done locally on the task experiencing the issue, like : https://github.com/kubernetes-sigs/kubespray/blob/0e91000a04edabcaae0590e8bb87f10e2542627a/roles/kubernetes/kubeadm/tasks/main.yml#L87-L96

Or if this affect several tasks, have another variables which is combined to form the full list at the relevant use sites.